### PR TITLE
Adding an outline schedule for Boston

### PR DESCRIPTION
**Pull request by glenrobson**

### CODE REVIEW

**Comment by glenrobson** [comment]
Branch aviliable for testing at https://preview.iiif.io/root/boston_outline

File: source/event/2020/boston/index.md [hunk 1]
```diff
@@ -32,6 +32,49 @@ Stay tuned to the [IIIF-Discuss][iiif-discuss] email list for announcements and
...
+            <td>Morning</td>
+            <td>&nbsp;</td>
+            <td>Workshops</td>
+            <td>Plenary lightning talks and parallel sessions</td>
+            <td>Parallel sessions</td>
+        </tr>
+        <tr>
+            <td>Afternoon</td>
+            <td>Showcase, MIT Stata Center</td>
+            <td>Conference plenary including Community updates and Lighning talks</td>
```
Comment (areichclir) [inline source/event/2020/boston/index.md]: Update language to: Conference plenary including community updates and lightning talks

File: source/event/2020/boston/index.md [hunk 2]
```diff
@@ -32,6 +32,49 @@ Stay tuned to the [IIIF-Discuss][iiif-discuss] email list for announcements and
...
+            <td>Parallel sessions</td>
+            <td>Closing lightning talks and panel sessions</td>
+        </tr>
+        <tr>
+            <td>Evening</td>
+            <td>&nbsp;</td>
+            <td>Conference Reception, MIT Museum</td>
+            <td>IIIF Consortium members Receiption, Harvard Natural History Museum</td>
```
Comment (areichclir) [inline source/event/2020/boston/index.md]: Update language to: IIIF Consortium members' reception, Harvard Natural History Museum

Comment (glenrobson) [reply source/event/2020/boston/index.md]: Should I remove the location?

Comment (areichclir) [reply source/event/2020/boston/index.md]: Yes please!

File: source/event/2020/boston/index.md [hunk 3]
```diff
@@ -61,16 +61,16 @@ The final program will continue to develop over the next few months but please s
...
+            <td>Conference plenary including community updates and lightning talks</td>
             <td>Parallel sessions</td>
             <td>Closing lightning talks and panel sessions</td>
         </tr>
         <tr>
             <td>Evening</td>
             <td>&nbsp;</td>
-            <td>Conference Reception, MIT Museum</td>
-            <td>IIIF Consortium members Receiption, Harvard Natural History Museum</td>
+            <td>Conference Reception</td>
```
Comment (areichclir) [inline source/event/2020/boston/index.md]: Update language to: Conference reception

**Pull request merged**